src/client/src/containers/About/index.tsx

```diff
@@ -38,6 +39,7 @@ const About = ({ versions, intl }: Props) => {
         <a
           className={styles.link}
           href="https://github.com/Microsoft/WebTemplateStudio"
```

Comment:
should this be hard coded into here?  I realize it's not your code but still...  (Consider a new task to fix this, not here)